packages/labs/react/src/create-component.ts

```diff
@@ -241,6 +243,22 @@ export function createComponent<
     tag = tagName;
   }
 
+  // Warn users when web components use reserved React properties
+  if (MODE === 'dev') {
+    for (const p in element.prototype) {
```

Comment:
Since the `reservedReactProperties size` is much smaller than the set of element properties, lets iterate over that and do an `in` check.

Reply:
Done!